src/Javabasic.java

```diff
@@ -1,6 +1,100 @@
+import java.util.Scanner;
 public class Javabasic {
 
-    public static void main(String[] args) {
-        
+
+                public static void remove() { // this method removes letter s from string
+            String sample = "This is very easy programme";  // sample string
+            sample = sample.replace("s", "");
+            System.out.print( "1. String without s will look like : " ); //printing output
```

Comment:
using String concatenation will make it more readable e.g. `System.out.print( "1. String without s will look like : " + sample  );`
another good option is string formatting  e.g. https://www.geeksforgeeks.org/java-string-format-examples/